pg2kinesis/__main__.py

```diff
@@ -32,8 +32,11 @@
               help='Attempt to on start create a the slot.')
 @click.option('--recreate-slot', default=False, is_flag=True,
               help='Deletes the slot on start if it exists and then creates.')
+@click.option('--change-kind', default='All',
+              type=click.Choice(['All', 'Update', 'Insert', 'Delete', 'Truncate']),
```

Comment:
nit, i'd call this --replicate-only and accept a string that has any combination of `audit`: 
- a(ll), 
- u(pdates) 
- i(nserts) 
- d(eletes) 
- t(runcates)

Reply:
basically, you can do something like:
```
replication_types = {
  rep_type[0]: rep_type for rep_type in [ 'all', 'update', 'insert', 'delete', 'truncate' ]
}

changes_to_replicate = [replication_types.get(op) for key in replicate_only]
if None in changes_to_replicate:
  raise Exception('Invalid replication type specified, must be one of 'audit')
```

Reply:
I'm sure there's a `@click` way to handle that

Reply:
Having dug into the code the reason it is called this is because other parts of the code reference a change 'kind' so this is where the name has come from. 

I agree it wouldnt have been my choice of name but I feel overhauling the whole thing would mean a pr wouldnt be accepted to the main repo. 

see:
https://github.com/shopkeep/pg2kinesis/blob/adding-change-kind-option/pg2kinesis/formatter.py#L91-L93

Furthermore being able to use any combination is a nice to have that we will address in a separate pr if needed.